cmd/flags.go

```diff
@@ -263,7 +251,11 @@ var disableFeatureFlags = []cliFlag{
 // checks if they are set in viper, and returns an error if they are not.
 func checkFlags(cmd *cobra.Command) error {
 	cmd.Flags().VisitAll(func(f *pflag.Flag) {
-		viper.BindPFlag(f.Name, f)
+		err := viper.BindPFlag(f.Name, f)
+		if err != nil {
+			fmt.Fprintf(os.Stderr, "Error binding flag %s: %v\n", f.Name, err)
```

Comment:
Is there something the end user should do here? Is this a configuration issue on the user's end or would this indicate a bug in the code? Maybe a better way to ask is - what should an end user _do_ with the information from this error?